sources/lib/reasoners/intervalCalculus.ml

```diff
@@ -1497,6 +1497,7 @@ let assume ~query env uf la =
          try
            match a with
            | L.Builtin(_, ((L.LE | L.LT) as n), [r1;r2]) ->
+             Steps.incr (Builtin);
```

Comment:
Is the `Builtin` step counter defined now ? i.e. do we have a specification as to what kinds of steps it counts ? because here it appears to deal with arithmetic ?

Reply:
This step counter is named `Builtin` because of the previous line (1499), it indeed deal with arithmetic. It can be rename to `Arith` if you prefere ?

Reply:
Well, first yes, a name more related to the fact that it deals with arithmetic would be better, but basically, it bugs me that we would have 3 different counters for arithmetic: Omega, Fourier, and builtin. Do we want/need to distinguish the three ? If so for what reasons (which could be spelled out in the doc of the steps module).

Reply:
We want to distinguish them because they have different multipliers.